examples/docs/solveQVSStls.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
```

Comment:
This example is too complicated. Make something like fixedAdrQvs.py or merge the two examples together. Generally speaking, an example should just show what are the minimal set of instructions to get a scheme to work